tests/UI/campaigns/functional/BO/02_orders/02_invoices/02_generateInvoiceByStatus.js

```diff
@@ -131,5 +131,19 @@ describe('BO - Orders - Invoices : Generate PDF file by status', async () => {
       const exist = await files.doesFileExist(filePath);
       await expect(exist).to.be.true;
     });
+
+    it('should choose one status, generate the invoice and check the file existence', async function () {
```

Comment:
@khouloudbelguith Is it normal the duplication with the previous test ?

Reply:
@Progi1984, I'm updating the automatic tests according to the scenario written and approved in the Jira.
The first test is selecting two statuses and the second is about selecting only one status.
Thanks!